scripts/run_nodes.sh

```diff
@@ -52,7 +52,7 @@ validator_ids_string="${validator_ids[*]}"
 validator_ids_string="${validator_ids_string//${IFS:0:1}/,}"
 
 echo "Bootstrapping chain for nodes 0..$((N_VALIDATORS - 1))"
-./target/release/aleph-node bootstrap-chain --base-path "$BASE_PATH" --account-ids "$validator_ids_string" --chain-type local > "$BASE_PATH/chainspec.json"
+./target/release/aleph-node bootstrap-chain --raw --base-path "$BASE_PATH" --account-ids "$validator_ids_string" --chain-type local > "$BASE_PATH/chainspec.json"
```

Comment:
Why we added `--raw` here? This script is used mostly for development purposes to my knowledge, and having non-raw chainspec makes debugging easier when it comes to e.g. adding new pallet

Reply:
But not having a raw chainspec also produces cryptic errors when you are trying to run it on two different versions.

Perhaps we could produce both?